src/components/UI/Tables/GenericTable/index.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import {
+  TableContainer,
+  Paper,
+  Table,
+  TableHead,
+  TableRow,
+  TableCell,
+  TableBody,
+} from "@mui/material";
+
+export interface IGenericTableColumn {
+  key: string;
```

Comment:
why do we need a key as a prop?

Reply:
i made it optional, i need it to render the columns